app/screens/table/table.android.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    ScrollView,
+} from 'react-native';
+import {makeStyleSheetFromTheme} from 'app/utils/theme';
+
+export default class Table extends React.PureComponent {
```

Comment:
Why two files ? they do not differ much is only the props for the nested ScrollView and the displayFlex style, I think combining this two files into one makes more sense and then use the `Platform` module

Reply:
Ah yep, sure thing

Will merge those back into a single file with two flex styles

Reply:
@avasconcelos114 you can do

```js
const getStyleSheet = makeStyleSheetFromTheme(() => {
    return {
        displayFlex: {
            ...Platform.select({
                android: {
                    flex: 1,
                },
                ios: {
                    flex: 0,
                },
            },
        },
    };
});
```

Reply:
Oh that's neat!

Thanks for sharing this one, didn't know about this alternative :)